### PR TITLE
[zh-cn] sync names components hello-minikube

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/overview/components.md
content/zh-cn/docs/concepts/overview/working-with-objects/names.md
content/zh-cn/docs/tutorials/hello-minikube.md

### CODE REVIEW

File: content/zh-cn/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -449,9 +457,37 @@ Minikube 有一组内置的{{< glossary_tooltip text="插件" term_id="addons" >
    ```
 
 <!--
-4. Disable `metrics-server`:
+1. Check the output from `metrics-server`:
 -->
-4. 禁用 `metrics-server`：
+4. 检查 `metrics-server` 的输出：
+
+    ```shell
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/hello-minikube.md:464]: Don't indent by 4 spaces here, indent by 3 spaces.

File: content/zh-cn/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -505,6 +541,15 @@ If you want to use minikube again to learn more about Kubernetes, you don't need
...
 如果你还想使用 Minikube 进一步学习 Kubernetes，那就不需要删除 Minikube。
 
+<!--
+## Conclusion
+
+This page covered the basic aspects to get a minikube cluster up and running. You are now ready to deploy applications.
+-->
+## 结论
+
+本页介绍了启动和运行 minikube 集群的基本方面，现在部署应用的准备工作已经完成。
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/hello-minikube.md:551]:
```suggestion
本页介绍了启动和运行 minikube 集群的基本知识，现在部署应用的准备工作已经完成。
```

**Review by windsonsea — COMMENTED** [review]
A nit.
/approve

**Pull request merged**